src/main/webapp/filesFragment.xhtml

```diff
@@ -221,8 +221,7 @@
         </p:column>
         <p:column class="col-file-metadata">
             
-            <ui:fragment rendered="#{fileMetadata.datasetVersion.released or ((fileMetadata.datasetVersion.deaccessioned or fileMetadata.datasetVersion.draft) 
-                                     and permissionServiceBean.on(DatasetPage.dataset).has('ViewUnpublishedDataset'))}">
```

Comment:
This call to the permissionServiceBean caused a lot of slowdown. The rendered tag wasn't useful anyways, if it was false it would stop the title of the file from showing up. If you can't view the unpublished dataset you can't get to the page anyways.